x-pack/plugins/index_management/public/sections/index_list/components/index_table/index_table.js

```diff
@@ -406,6 +416,21 @@ export class IndexTableUi extends Component {
                   onChange={this.onFilterChanged}
                 />
               </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiButton
+                  color="secondary"
+                  onClick={() => {
+                    loadIndices();
+                    filterChanged(EuiSearchBar.Query.parse(''));
```

Comment:
Why do we want to clear the filters here? This seems like surprising behavior.

Reply:
the reload is intended to pick up new indices.  if we don't clear the filters, they may not see the new indices.

Reply:
removed clearing filters